bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/TextImpl.java

```diff
@@ -50,4 +58,16 @@ public String getValue() {
     public boolean isRichText() {
         return textIsRich;
     }
+
+    @Override
+    @NotNull
+    protected FormComponentData getComponentData() {
```

Comment:
why this needs to be override?

Reply:
To add one extra property in case of image. i.e. text
I've added all the property for each component in the JIRA description.
https://jira.corp.adobe.com/browse/FORMS-5839

Reply:
removed